lib/xccdf2inspec/xccdf2inspec.rb

```diff
@@ -162,6 +183,15 @@ def print_benchmark_info
   # @todo Allow for the user to pass in a hash for the desired mapping of text
   # values to numbers or to override our hard coded values.
   #
+  def mapperFunc(diacap)
+    xlsx = Spreadsheet.open "./DIACAP.xls"
```

Comment:
The diacap xls file should be a variable that comes form the a flag on the command line like the other input files correct?
 We should not hard code any data as we then have to make sure they understand that we hard coded something.